App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift

```diff
@@ -0,0 +1,194 @@
+// system
+import SwiftUI
+
+// Internal
+
+// 3rd
+import OptionalAPI
+
+
+protocol SizingFrameType {
```

Comment:
Maybe this should be moved to `Utilities`? :)

Reply:
Moved